code/src/cljs/sixsq/nuvla/ui/notifications/views.cljs

```diff
@@ -308,7 +309,9 @@
                              {:key "network tx" :text "Network Tx GiB" :value utils/network-tx}
                              ]
    "infrastructure-service" [{:key "status" :text "status" :value utils/status}]
-   "data-record"            [{:key "content-type" :text "content-type" :value utils/content-type}]})
+   "data-record"            [{:key "content-type" :text "content-type" :value utils/content-type}]
```

Comment:
Missing translation

Reply:
"Published for App Bouquet" and "Published for Deployments" are criteria definitions. Agreed to do the translation later.